homeassistant/components/jewish_calendar/binary_sensor.py

```diff
@@ -20,10 +22,38 @@
 
 from . import DOMAIN
 
-BINARY_SENSORS = BinarySensorEntityDescription(
-    key="issur_melacha_in_effect",
-    name="Issur Melacha in Effect",
-    icon="mdi:power-plug-off",
+
+@dataclass
+class JewishCalendarBinarySensorMixIns(BinarySensorEntityDescription):
+    """Binary Sensor description mixin class for Jewish Calendar."""
+
+    is_on: Callable = lambda _: False
```

Comment:
```suggestion
    is_on: Callable[..., bool] = lambda _: False
```
I think defining the return value here_could_ remove the need to perform casting later on.